client/js/components/faq/DpFaqItem.vue

```diff
@@ -15,25 +15,25 @@
  --><div class="layout__item u-4-of-12  u-pt-0_125">
       <dp-multiselect
         v-if="availableGroupOptions.length > 1"
-        track-by="id"
-        multiple
-        :options="availableGroupOptions"
         :allow-empty="false"
-        :custom-label="option =>`${option.title}`"
+        :custom-label="props =>`${props.option.title}`"
         data-cy="selectedGroups"
+        multiple
+        :options="availableGroupOptions"
+        track-by="id"
         :value="selectedGroups"
         @input="selectGroups">
-        <template v-slot:option="{ option }">
-          <span>{{ option.title }}</span>
+        <template v-slot:option="{ props }">
```

Comment:
Do we really need to pass in the whole `props` object here? See question in other PR.

Reply:
Yes, because otherwise we can't access the `remove` function. Or do you  have an idea about that?

Reply:
no, not at the first glance.